kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -21,10 +22,24 @@
     undo_aliases,
 )
 from pyk.kcfg import KCFG
-from pyk.ktool import KPrint
+from pyk.ktool import KPrint, KProve
 from pyk.prelude import Bool, mlAnd
 
 
+def KProve_prove_claim(  # noqa: N802
```

Comment:
It is safe to replace `KProve.prove_claim` by this, right?

Reply:
Well, not really. This is just a wrapper around KProve.prove_claim, so it assumes all the functionality of that one.

Reply:
I mean after inlining (so that there is no dependency to the original method), could we exclusively use this one instead? If so, we should do the refactoring.

Reply:
Hmmm, not quite sure. This one assumes you won't do `--dry-run`. But yes, it should be refactored/upstreamed. It's listed: https://github.com/runtimeverification/ksummarize/issues/82